### DISCUSSION TITLE
Battery buttons and ECAM

### DISCUSSION THREAD

**Discussion by LukasGNN** [flybywiresim/aircraft#3906] [Q&A] [answered]
hi, i managed to program (in spad.next) battery by using LVAR. I read in data monitor that there are used 2 variables:XMLVAR_MOMENTARY_PUSH_OVHD_ELEC_BAT1_PRESSED and A32NX_OVHD_ELEC_BAT_10_PB_IS_AUTO. When i press  battery button manually in sim, ECAM shows battery status ON. But if i use programmed button with mentioned above variables, battery is ON, but ECAM still shows battery OFF. Does anyone know how to get correct information in ECaM. Do i miss some LVAR variable??

**Comment by hotshotp** [comment, marked as answer]
use dev version with new elec ecam and elec system

**Reply by LukasGNN** [reply]
it works now :) 
One more question: 
I can controll on/off function on BAT 2 and OFF on BAT 1. How to controll ON on BAT1?? 
I use A32NX-OVHD_ELEC_BAT10_PB_IS_AUTO  and ELECTRICAL_BUS_CONNECTION_TOGGLE: 6, but this works only as OFF function

On bat 2 ON is electrical_bus_to_bus_connection_toggle:6 and buss connection on:6 to 1. 
Any idea how to switch on BAT 1??
